shared/model.py

```diff
@@ -26,7 +26,8 @@
 
 db = SQLAlchemy(app)
 
-MAX_WEEK_KEEP_RECORDS = app.config.get("MAX_WEEK_KEEP_RECORDS", 5)
+MAX_DAYS_KEEP_UNFILTERED_RECORDS = app.config.get("MAX_DAYS_KEEP_UNFILTERED_RECORDS", 0)
```

Comment:
The current default is 35 (5 weeks). Are you proposing to change the default? I am open to changing the default, but I would like to see justification.

Reply:
This is going to be based on what we think would be the most common case, since we do not know lets keep the 35 days one.